docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -2,8 +2,7 @@
 Getting Started with Galaxy
 ===========================
 
-Galaxy workflow support in Dockstore is new and a preview of
-more full featured support to come.
+Dockstore has recently launched the Galaxy workflow support and the _`Launch with Galaxy <../launch-with/galaxy-launch-with>`__ tutorial elaborates different ways to launch a Galaxy workflow.
```

Comment:
I suggest you rephrase to not have "recently launched" -- otherwise somebody is going to have come back here a year from now and remove "recently" (and remember to do it!).

Reply:
Link not showing up correctly in https://docs.dockstore.org/en/feature-update-launchwithgalaxy/getting-started/getting-started-with-galaxy.html